cypress/integration/property/show.spec.js

```diff
@@ -382,6 +382,35 @@ describe('Show property', () => {
     })
   })
 
+  context(
+    'Does not display TMO name if TMO is London Borough of Hackney',
+    () => {
+      beforeEach(() => {
+        // Stub request with property response
+        cy.fixture('properties/propertyWithTmo.json')
+          .then((property) => {
+            property.tmoName = 'London Borough of Hackney'
+            cy.intercept(
+              { method: 'GET', path: '/api/properties/00012345' },
+              { body: property }
+            )
+          })
+          .as('property')
+
+        cy.visit('/properties/00012345')
+        cy.wait('@property')
+      })
+
+      it('shows property address in the heading', () => {
+        cy.get('.lbh-heading-h1').contains('12 Test Street')
+      })
```

Comment:
I don't think we need this test as we've already got it covered on https://github.com/LBHackney-IT/repairs-hub-frontend/pull/527/commits/74bcb156e1ea97b33dcc30d683ecd00e48e599aa#diff-5635bf6f4285741ca6764002d84a6e3bb14623239043e1be10e10a1a70e78e48R373 and we don't expect that to change based on the setup for this test.